src/parser/shaman/elemental/modules/talents/Ascendance.js

```diff
@@ -35,36 +52,94 @@ class Ascendance extends Analyzer {
     return this.rawUpdate;
   }
 
+  get averageLavaBurstCasts() {
+    return this.numCasts[SPELLS.LAVA_BURST.id]/this.numCasts[SPELLS.ASCENDANCE_TALENT_ELEMENTAL.id];
```

Comment:
Please check for division by zero in case player didn't cast Ascendance
```suggestion
    return (this.numCasts[SPELLS.LAVA_BURST.id] / this.numCasts[SPELLS.ASCENDANCE_TALENT_ELEMENTAL.id]) || 0;
```